### ISSUE TITLE
dropped HTTPS support for legacy

### ISSUE THREAD

**Issue by gingerlime** [fixerAPI/fixer#119]
We're suddenly seeing these (hidden) errors from fixer

```ruby
irb(main):001:0>     response = RestClient.get(
irb(main):002:1*       "https://data.fixer.io/api/latest",
irb(main):003:1*       :params => {:base => "USD", :access_key => fixer_api_key}
irb(main):004:1>     )
=> <RestClient::Response 200 "{\"success\":...">
irb(main):005:0> JSON.parse(response)
=> {"success"=>false, "error"=>{"code"=>105, "type"=>"https_access_restricted", "info"=>"Access Restricted - Your current Subscription Plan does not support HTTPS Encryption."}}
```

We're on the free legacy plan ... We can switch to HTTP obviously. Those conversion rates are not really a secret, but our api key kinda is, isn't it? Are you actually encouraging people to leak their own API keys to your system??

We don't have a problem paying for a good service (in fact, I upgraded to a paid account), but this feels a bit like bait-and-switch, I have to say... Especially since you guys dropped the open API a while back which also caused us issues until we signed for the legacy account. So it's not the first time :-/

Returning an error inside a 200 seems also a bit weird and I was lucky to debug this by chance and find out it was doing this...

**Comment by sayst-matt** [comment]
Yes, we have the same problem on the legacy plan as a non-profit. 

Is this a permanent change, and was this change announced anywhere?

**Comment by gingerlime** [comment]
They sent an email (which we missed):

![Screen Shot 2019-07-02 at 15 54 39](https://user-images.githubusercontent.com/524274/60518422-cba9fa80-9ce1-11e9-8e32-cc6e52d1cb4e.png)

**Comment by sayst-matt** [comment]
Ah, unfortunately it was captured by my spam blocker so I've only just seen it. Thank you!
